pulseapi/entries/test_views.py

```diff
@@ -186,10 +189,12 @@ def test_post_entry_with_mixed_creators(self):
             '/api/pulse/entries/',
             data=self.generatePostPayload(data=payload)
         )
-        creatorList = json.loads(
+        creator_list = json.loads(
             str(self.client.get('/api/pulse/creators/').content, 'utf-8')
         )
-        self.assertEqual(creatorList, ['Pomax', 'Alan'])
+        db_creator_list = [c.name for c in self.creators]
+        db_creator_list.extend(creators)
```

Comment:
same question as above really: why the extend? does the self-extraction not cover it? (if not: should we simply reload `self` from the db?